behave/docker-compose.yml

```diff
@@ -148,6 +162,29 @@ services:
       STATIC_FILES_BACKEND: local
       ADMIN_PASSWORD: cla_admin
       ALLOWED_HOSTS: "*"
+      NON_ROTA_START_TIME_HR: 1
```

Comment:
what happens if these tests run after 22:00? If it is 23:00, is this then out of hours?

Reply:
Yeah it'll classify it out of hours, there is no way around this without re-writing CLA_Backend as the datetime is preset to accept 0 minutes so we cannot make the end time 23:58 to 23:59, 23 is the latest as datetime won't accept 24

Reply:
can we at least make it 0 to 23?